boot/freeldr/freeldr/arch/realmode/helpers_all.inc

```diff
@@ -0,0 +1,81 @@
+/*
+ * PROJECT:     FreeLoader
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Real mode helper code for all architectures
+ * COPYRIGHT:   Copyright 2024 Daniel Victor <ilauncherdeveloper@gmail.com>
+ */
+
+Relocator16SS:
+    .word 0
+Relocator16EAX:
+    .long 0
+Relocator16ESP:
+    .long 0
+
+cs_writestr:
+    push ax
+    push ds
+    mov ax, cs
+    mov ds, ax
+
+    call writestr
+    pop ds
+    pop ax
+    ret
+
+GetBaseAddress:
```

Comment:
The base address of what?

Reply:
the current code base address

Reply:
the `GetCurrentBaseAddress` function name help for devs understand?